providers/sendgrid/src/lib/sendgrid.provider.ts

```diff
@@ -7,10 +7,13 @@ import {
   ICheckIntegrationResponse,
   CheckIntegrationResponseEnum,
   IEmailEventBody,
+  IAttachmentOptions,
 } from '@novu/stateless';
 
 import { MailDataRequired, MailService } from '@sendgrid/mail';
 
+type AttachmentJSON = MailDataRequired['attachments'][0];
```

Comment:
No idea why but the AttachmentJSON is not exported from '@sendgrid/mail', created this workaround so we won't use any in the code.